crm_claim_type/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'CRM Claim Types',
+    'category': 'Customer Relationship Management',
+    'summary': 'Claim types for CRM',
+    'author': 'Vauxoo',
```

Comment:
@osvalr Like this https://github.com/OCA/rma/pull/32/files#diff-e4f8bc2bd35f91a5974c441fc33cdd7dR75